shared/js/background/atb.es6.js

```diff
@@ -20,9 +20,14 @@ const ATB = (() => {
     // regex to match ddg urls to add atb params to.
     // Matching subdomains, searches, and newsletter page
     const regExpAboutPage = /^https?:\/\/([\w-]+\.)?duckduckgo\.com\/(\?.*|about#newsletter)/
+    const matchPage = /^https?:\/\/([\w-]+\.)?duckduckgo.com\/\?/
```

Comment:
nit: We could probably drop the `?` on `https`? -- I don't think we'd ever have someone install the extension from an `http` domain